UpsertedProfileOrchestrator/handler.ts

```diff
@@ -49,6 +49,9 @@ export const OrchestratorInput = t.intersection([
     updatedAt: UTCISODateFromString
   }),
   t.partial({
+    // TODO: name field from partial to required after complete rollout of
+    // IOPID-1444 task
+    name: t.string,
```

Comment:
We should ensure that the new `name` property is added to all inputs of `UpsertedProfileOrchestrator`.

Reply:
done in https://github.com/pagopa/io-functions-app/pull/304/commits/231f4842a7f9be6a3bf98b1577a31eacb377f31b